cf-nodejs-logging-support-core/log-core.js

```diff
@@ -572,15 +591,16 @@ var writeCustomFields = function (logObject, logger, additionalFields) {
             value = stringifySafe(value);
         }
 
-        if (registeredCustomFields.includes(key)) {
-            customFields[key] = value;
-        } else if (logObject[key] == null && key != "custom_fields") {
+        if(registeredCustomFields.includes(key) && defaultCustomEnabled) {
             logObject[key] = value;
         }
+        if (registeredCustomFields.includes(key) && cfCustomEnabled) {
```

Comment:
This changes the previous logic under which a custom field could only either in `logObject` or in `customFields`. However, this syntax is much closer to the _retained fields_ in cf-java-logging-support.

Reply:
This is needed to provide both formats if people bind to  both app and cloud logging